app/models/page_view.rb

```diff
@@ -6,6 +6,12 @@ class PageView < ApplicationRecord
 
   before_create :extract_domain_and_path
 
+  ##
+  # Call TweetTagRefreshStatsJob with probability of 1/N
+  # Not something we need to perform for every time a PageView is created.
+  # If we find we want refresh this more often, we can lower what N is
+  after_create :tweet_tag_refresh_stats, if: proc { rand(30).eql?(1) }
```

Comment:
I'm not a superfan of the idea of tying the creation of a PageView with the TweetTag refresh behavior in the model.

If you think about it this logic shouldn't be here. page view objects shouldn't know anything about tweet tags.

I wonder if we could extract this in the PageViewController. Basically after the PageView object is created, we call
TweetTagRefreshStatsJob.perform_later(article.id) guarded by the rand call.